packages/swapper/src/types.ts

```diff
@@ -140,8 +140,8 @@ type TradeQuoteBase = {
   rate: string // top-level rate for all steps (i.e. output amount / input amount)
   receiveAddress: string
   receiveAccountNumber?: number
-  potentialAffiliateBps: string | undefined // undefined if affiliate fees aren't supported by the swapper
-  affiliateBps: string | undefined // undefined if affiliate fees aren't supported by the swapper
+  potentialAffiliateBps: string | undefined // even if the swapper does not support affiliateBps, we need to zero-them out or view-layer will be borked
+  affiliateBps: string | undefined // // even if the swapper does not support affiliateBps, we need to zero-them out or view-layer will be borked
```

Comment:
make required and i think the comment in cowswap quote is sufficient imo
```suggestion
  potentialAffiliateBps: string
  affiliateBps: string
```

Reply:
absolutely - 6252df3c10